### PR TITLE
Feature navbar

### PR DESCRIPTION
**Pull request by dallinbjohnson**
## Brief description

reformatted the scss.

## Developer Certificate of Origin

- [x] I certify that these changes according to the Developer Certificate of Origin 1.1 as described at <https://developercertificate.org/>.

## Sample pictures

...

## Further details

...

### CODE REVIEW

**Comment by dallinbjohnson** [comment]
I have never worked with SCSS before and I looked at the resp mixin and I am not sure how to integrate it into my code.  I did everything that I understood the rest is a little confusing to me.  sorry. @koester

**Comment by koester** [comment]
Being new or not knowing  something is not a problem at all. We've all been there. When I'm at the office I'll edit this comment and explain how to use the resp mixing (or scss mixins in general) and give you some examples.

File: src/components/_navbar.scss [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    transform: rotate(-45deg) translate(-9px, 7px);
+  }
+  .collapsible input[id^=collapsible]:checked + button .bar2 {
+    opacity: 0;
+  }
+  .collapsible input[id^=collapsible]:checked + button .bar3 {
+    -webkit-transform: rotate(45deg) translate(-8px, -9px);
+    transform: rotate(45deg) translate(-8px, -9px);
+  }
+}
```
Comment (koester) [inline src/components/_navbar.scss]: I can see a pattern here. Is there a way to further nest those selectors and reduce the duplicated code?

File: src/components/_navbar.scss [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+nav.split-nav {
+  justify-content: space-between;
+}
+nav.fixed {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+}
```
Comment (koester) [inline src/components/_navbar.scss]: Different Classes for the same parent? How could you nest them together so you only have one `nav` with everything else nested inside?

**Comment by rhyneav** [comment]
@koester where can I read more of your writing? 

@TotomInc I think it looks good! Maybe we can play around with different border thicknesses, but overall I like the look.

**Comment by rhyneav** [comment]
@koester thoughtsofkoester.com

**Comment by koester** [comment]
@rhyneav Oh shieeet, thats one hell of  a domain name for a blog. Now I'm really thinking of starting a blog. Just one small problem remains.. what the hell should I blog about? :thinking:

@dallinbjohnson Do you need further help? If there are still questions left feel free to ask.

**Comment by dallinbjohnson** [comment]
Sorry I have not had time to work on this I am working and going to school. It's getting close to finals. I will make time to work on it though.  Everything that you taught me was great it really helped me understand.

**Comment by dallinbjohnson** [comment]
How do I create another pull request with the changes that I have made to my feture-navbar?

**Comment by dallinbjohnson** [comment]
what are the checks that are not successful?  and is there anything else I need to change to make my code so it can be merged? @rhyneav

**Comment by rhyneav** [comment]
I think the checks are giving an error because of the way npm is being installed in the test runner. I'm working on the fix in another branch. I don't think they're breaking from anything you're adding, don't worry!

Before getting your code merged in, make sure you are working off of the develop branch. With this, in Github, request to merge into the develop branch (rather than master). Also, @koester gave some great advice about DRYing up your code. Could you refactor your `_navbar.scss` to what he recommended?

**Comment by dallinbjohnson** [comment]
It should be fixed now! let me know if there are any more changes I need to make. @rhyneav

**Comment by rhyneav** [comment]
Looks great @dallinbjohnson thank you for taking care of those edits! Since this PR and #145 are identical, I'll merge #145 into develop since it is already pointing at it.

Great work, very excited to have a navbar added to PaperCSS!

**Pull request closed without merging**